scripts/analysis/analyze_original_vocabulary.py

```
@@ -0,0 +1,319 @@
+
+
+def extract_raw_response_data(raw_response: str) -> Dict[str, Any]:
+    """Extract data from raw_response JSON string."""
+    try:
+        # Find the JSON part in the raw response
+        start = raw_response.find("{")
+        end = raw_response.rfind("}") + 1
+        if start != -1 and end != -1:
+            json_str = raw_response[start:end]
+            return json.loads(json_str)
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)